src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
+$font-family: Roboto, sans-serif;
+$text-color: #060b35;
+$secondary-color: #616070;
+
 body {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  justify-content: center;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.product {
+  font-family: $font-family;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
```

Comment:
it is a content image - you added it using `img` tag - no need to set background image too